### ISSUE TITLE
build also for normal React component require

### ISSUE THREAD

**Issue by forresto** [the-grid/ed#230]
Will need some refactoring, since the top-level `Ed` isn't a React component (it's the store and mounter and dataflow).

**Comment by netlemur** [comment]
@forresto anything I can do to help?

It would be great if we can just use

``` js
import TheGridEd from '@the-grid/ed'

export default class PostEditor extends React.Component {
  render() {
    return (
      <TheGridEd initialContent={...} onChange={...} ... />
    )
  }
}
```

and re-use the react version from webapp. That would easily save a few hundred kb in the webapp build.

**Comment by forresto** [comment]
⚛>⚛>⚛>⚛...

**Comment by forresto** [comment]
@netlemur having trouble getting in there... mounted is supposed to be the instance of the component (where you can do stuff like `getContent()`), but is coming back `null`. https://facebook.github.io/react/docs/top-level-api.html#reactdom.render says that's the case for stateless components, but App is a class not a stateless function component.

Is it b/c I'm not using jsx?

```
import App from './components/app'

import React from 'react'
import ReactDOM from 'react-dom'


export default App

export function mountApp (container, props) {
  // Setup main DOM structure
  const app = React.createElement(App, props)
  const mounted = ReactDOM.render(app, container)
  return mounted
}

export function unmountApp (container) {
  ReactDOM.unmountComponentAtNode(container)
}
```

**Comment by forresto** [comment]
Caused by, in app.js, `export default React.createFactory(App)` ... exporting the class instead of the factory fixes it. Why did I need those factories everywhere?

**Comment by forresto** [comment]
@netlemur caveat, webapp will need to copy widget iframes. I'll add a `widgetRootPath` property or something.

**Comment by forresto** [comment]
Progress on this, but wondering about the build vs `package.main` ... webapp doesn't want to build all of ed + prosemirror, right? But also the build shouldn't include React. Any Webpack examples?

**Comment by forresto** [comment]
@netlemur take a quick look before I release it? ☝️

**Comment by forresto** [comment]
Released with v0.15.0, can tweak from here.
